exercises/ledger/canonical-data.json

```
@@ -0,0 +1,240 @@
+      "description": "overlong description is truncated",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "en_US",
+        "entries": [
+          {
+            "date": "2015-01-01",
+            "description": "Freude schoner Gotterfunken",
+            "amountInCents": -123456
+          }
+        ]
+      },
+      "expected": [
+        "Date       | Description               | Change       ",
+        "01/01/2015 | Freude schoner Gotterf... |   ($1,234.56)"
+      ]
+    },
+    {
+      "uuid": "67318aad-af53-4f3d-aa19-1293b4d4c924",
+      "description": "euros",
+      "property": "formatEntries",
+      "input": {
+        "currency": "EUR",
+        "locale": "en_US",
+        "entries": [
+          {
+            "date": "2015-01-01",
+            "description": "Buy present",
+            "amountInCents": -1000
+          }
+        ]
+      },
+      "expected": [
+        "Date       | Description               | Change       ",
+        "01/01/2015 | Buy present               |      (€10.00)"
+      ]
+    },
+    {
+      "uuid": "bdc499b6-51f5-4117-95f2-43cb6737208e",
+      "description": "dutch locale",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "nl_NL",
+        "entries": [
+          {
+            "date": "2015-03-12",
+            "description": "Buy present",
+            "amountInCents": 123456
+          }
+        ]
+      },
+      "expected": [
+        "Datum      | Omschrijving              | Verandering  ",
+        "12-03-2015 | Buy present               |   $ 1.234,56 "
+      ]
+    },
+    {
+      "uuid": "86591cd4-1379-4208-ae54-0ee2652b4670",
+      "description": "dutch locale and euros",
+      "property": "formatEntries",
+      "input": {
+        "currency": "EUR",
+        "locale": "nl_NL",
+        "entries": [
+          {
+            "date": "2015-03-12",
+            "description": "Buy present",
+            "amountInCents": 123456
+          }
+        ]
+      },
+      "expected": [
+        "Datum      | Omschrijving              | Verandering  ",
+        "12-03-2015 | Buy present               |   € 1.234,56 "
+      ]
+    },
+    {
+      "uuid": "876bcec8-d7d7-4ba4-82bd-b836ac87c5d2",
+      "description": "dutch negative number with 3 digits before decimal point",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "nl_NL",
+        "entries": [
+          {
+            "date": "2015-03-12",
+            "description": "Buy present",
+            "amountInCents": -12345
+          }
+        ]
+      },
+      "expected": [
+        "Datum      | Omschrijving              | Verandering  ",
+        "12-03-2015 | Buy present               |    $ -123,45 "
+      ]
+    },
+    {
+      "uuid": "29670d1c-56be-492a-9c5e-427e4b766309",
+      "description": "american negative number with 3 digits before decimal point",
```

```suggestion
      "description": "American negative number with 3 digits before decimal point",
```

Reason:  As a proper pronoun, but also it could be argued this is a sentence, and the first letter should be capitalized.

Same argument for "dutch" on line 201 and the prior use, in terms of "proper pronoun"

---

Applied. Thanks!